### PR TITLE
MES-6919: journal loading spinner bug

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

set isLoading to false upon success of load tests

## Checklist

- [ ] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/store/journal/journal.reducer.ts [hunk 1]
```diff
@@ -101,6 +101,7 @@ export const journalReducer = createReducer(
     completedTests,
   }): JournalModel => ({
     ...state,
+    isLoading: false,
```
Comment (andrewsetterfield) [inline src/store/journal/journal.reducer.ts:104]: wasnt brought over from des3

File: src/app/pages/journal/journal.page.ts [hunk 2]
```diff
@@ -231,6 +231,7 @@ export class JournalPage extends BasePageComponent implements OnInit {
     }
     if (this.pageRefresher) {
       this.pageRefresher['detail'].complete();
+      this.pageRefresher = null;
```
Comment (andrewsetterfield) [inline src/app/pages/journal/journal.page.ts:238]: prevents page flashing when navigate through days

File: src/app/pages/journal/journal.page.html [hunk 3]
```diff
@@ -5,7 +5,7 @@
       <mes-back-button routerLink="DashboardPage"></mes-back-button>
     </ion-buttons>
     <ion-buttons slot="end">
-      <ion-button (click)="refreshJournal()">
+      <ion-button (click)="refreshJournal()" [disabled]="pageState.isLoading$ | async">
```
Comment (andrewsetterfield) [inline src/app/pages/journal/journal.page.html:8]: disable button until loading complete

File: .nvmrc [hunk 4]
```diff
@@ -1 +1 @@
-10.13.0
\ No newline at end of file
+14.17.0
```
Comment (andrewsetterfield) [inline .nvmrc:1]: added so can use 'nvm use' to swap to specific node

File: angular.json [hunk 5]
```diff
@@ -195,4 +196,4 @@
       "styleext": "scss"
     }
   }
-}
+}
```
Comment (Cullima) [inline angular.json]: my fault but could you add back in?

File: angular.json [hunk 6]
```diff
@@ -185,7 +185,8 @@
     }
   },
   "cli": {
-    "defaultCollection": "@ionic/angular-toolkit"
+    "defaultCollection": "@ionic/angular-toolkit",
+    "analytics": false
```
Comment (tomcrawleyy) [inline angular.json]: revert this if you can

**Review by tomcrawleyy — APPROVED** [review]

**Review by Cullima — APPROVED** [review]

**Pull request merged**